packages/api/src/external/carequality/organization-conversion.ts

```diff
@@ -1,33 +1,33 @@
 import { uuidv7 } from "@metriport/core/util/uuid-v7";
 import { XCPDGateway } from "@metriport/ihe-gateway-sdk";
-import { Config } from "../../shared/config";
-import { getOidsWithIHEGatewayV2Enabled } from "../aws/app-config";
+import { getOidsWithIHEGatewayV2Enabled, isIHEGatewayV2EnabledForCx } from "../aws/app-config";
 import { CQOrgBasicDetails } from "./command/cq-directory/search-cq-directory";
 
-export async function cqOrgsToXCPDGateways(cqOrgs: CQOrgBasicDetails[]): Promise<{
+export async function cqOrgsToXCPDGateways(
+  cqOrgs: CQOrgBasicDetails[],
+  cxId: string
+): Promise<{
   v1Gateways: XCPDGateway[];
   v2Gateways: XCPDGateway[];
 }> {
   const v1Gateways: XCPDGateway[] = [];
   const v2Gateways: XCPDGateway[] = [];
-  const iheGatewayV2OIDs: string[] = Config.isDev()
-    ? Config.getOidsWithIHEGatewayV2Enabled().split(",")
-    : await getOidsWithIHEGatewayV2Enabled();
+  const iheGatewayV2OIDs: string[] = await getOidsWithIHEGatewayV2Enabled();
```

Comment:
nit the typing isnt necassary because the function has a return type which will be reflected in the varaible